### DISCUSSION TITLE
Ability to change spatial sound using right click like windows sound tray icon.

### DISCUSSION THREAD

**Discussion by tembu007** [File-New-Project/EarTrumpet#1158] [Ideas]
I think this would be a good thing to implement because windows always turns off spatial sound on my device and having both the system tray icon and eartrumpet is causing clutter. I have attached a picture of what I am talking about.
![image](https://user-images.githubusercontent.com/50773853/169042622-efc12580-8837-47b7-a124-8c32352a914a.png)

**Comment by B3H2** [comment]
That's the only reason I cant take the windows volume out of may tray!!!!!!!!

Please!!!!!!!!!!!!!!!!!!
